xbmc/platform/darwin/tvos/MainController.mm

```diff
@@ -126,28 +126,29 @@ - (void)sendButtonPressed:(int)buttonId
     CLog::Log(LOGDEBUG, "ERROR mapping customcontroller action. CustomController: %s %i", "SiriRemote", buttonId);
   }
 }
-//--------------------------------------------------------------
-//--------------------------------------------------------------
-- (void)sendKeyDownUp:(XBMCKey)key
-{
-  XBMC_Event newEvent = {0};
-  newEvent.key.keysym.sym = key;
-
-  newEvent.type = XBMC_KEYDOWN;
-  CWinSystemTVOS* winSystem(dynamic_cast<CWinSystemTVOS*>(CServiceBroker::GetWinSystem()));
-  winSystem->MessagePush(&newEvent);
 
-  newEvent.type = XBMC_KEYUP;
-  winSystem->MessagePush(&newEvent);
-}
-- (void)sendKeyDown:(XBMCKey)key
-{
-  XBMC_Event newEvent = {0};
-  newEvent.type = XBMC_KEYDOWN;
-  newEvent.key.keysym.sym = key;
-  CWinSystemTVOS* winSystem(dynamic_cast<CWinSystemTVOS*>(CServiceBroker::GetWinSystem()));
-  winSystem->MessagePush(&newEvent);
-}
+//--------------------------------------------------------------
+//- (void)sendKeyDownUp:(XBMCKey)key
+//{
+//  XBMC_Event newEvent = {0};
+//  newEvent.key.keysym.sym = key;
+//
+//  newEvent.type = XBMC_KEYDOWN;
+//  CWinSystemTVOS* winSystem(dynamic_cast<CWinSystemTVOS*>(CServiceBroker::GetWinSystem()));
+//  winSystem->MessagePush(&newEvent);
+//
+//  newEvent.type = XBMC_KEYUP;
+//  winSystem->MessagePush(&newEvent);
+//}
+
+//- (void)sendKeyDown:(XBMCKey)key
+//{
+//  XBMC_Event newEvent = {0};
+//  newEvent.type = XBMC_KEYDOWN;
+//  newEvent.key.keysym.sym = key;
+//  CWinSystemTVOS* winSystem(dynamic_cast<CWinSystemTVOS*>(CServiceBroker::GetWinSystem()));
+//  winSystem->MessagePush(&newEvent);
+//}
```

Comment:
Can we just drop all the above completely in this commit instead?

Reply:
sure